Limiting hosts per line works for first line, but after that each item goes on its own line

This isn't a huge deal, but probably not what's intended.

Using the artifacts in https://github.com/ddev/ddev/pull/4805 and running the manual test there, 
```
for item in 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15; do 
  sudo ddev hostname a$item.ddev.site 127.0.0.1
done
```
You'll see the results are that the first hosts line goes to 8, but after that they are all on separate lines. 

```
127.0.0.1 localhost junk.ddev.site a1.ddev.site a2.ddev.site a3.ddev.site a4.ddev.site a5.ddev.site a6.ddev.site
127.0.0.1 a15.ddev.site
127.0.0.1 a14.ddev.site
127.0.0.1 a13.ddev.site
127.0.0.1 a12.ddev.site
127.0.0.1 a11.ddev.site
127.0.0.1 a10.ddev.site
127.0.0.1 a9.ddev.site
127.0.0.1 a8.ddev.site
127.0.0.1 a7.ddev.site
```

Code from https://github.com/rfay/ddev/blob/b71df9536b015854d9ee3cc7829861ef34e0e909/pkg/ddevapp/hostname_mgt.go#L135-L146 is 
```
	hosts, err := goodhosts.NewCustomHosts(osHostsFilePath)

	if err != nil {
		return err
	}
	err = hosts.Add(ip, name)
	if err != nil {
		return err
	}
	hosts.HostsPerLine(8)
	err = hosts.Flush()
	return err
```

Sadly, it has a panic now, 
```
panic: runtime error: index out of range [18] with length 1

goroutine 1 [running]:
github.com/goodhosts/hostsfile.(*Hosts).Add(0x140000ea000, {0x16f82faf4, 0x9}, {0x140000c0130?, 0x1, 0x1})
	/Users/rfay/workspace/ddev/vendor/github.com/goodhosts/hostsfile/hosts.go:179 +0x538
github.com/ddev/ddev/pkg/ddevapp.AddHostEntry({0x16f82fae6, 0xd}, {0x16f82faf4, 0x9})
```

--- a/hosts.go
+++ b/hosts.go
@@ -176,8 +176,8 @@ func (h *Hosts) Add(ip string, hosts ...string) error {
 	} else {
 		// add new host to the first one we find
 		loc := position[len(position)-1] // last element
-		hostsCopy := make([]string, len(h.Lines[position[loc]].Hosts))
-		copy(hostsCopy, h.Lines[position[loc]].Hosts)
+		hostsCopy := make([]string, len(h.Lines[loc].Hosts))
+		copy(hostsCopy, h.Lines[loc].Hosts)
 
 		for _, addHost := range hosts {
 			if h.Has(ip, addHost) {
@@ -189,10 +189,10 @@ func (h *Hosts) Add(ip string, hosts ...string) error {
 			}
 
 			hostsCopy = append(hostsCopy, addHost)
-			h.hosts.add(addHost, position[loc])
+			h.hosts.add(addHost, loc)
 		}
-		h.Lines[position[loc]].Hosts = hostsCopy
-		h.Lines[position[loc]].RegenRaw()
+		h.Lines[loc].Hosts = hostsCopy
+		h.Lines[loc].RegenRaw()
 	}
 
 	return nil

